Tasks/Common/PowershellHelpers/Helpers.ps1

```diff
@@ -43,15 +48,22 @@ function Invoke-ActionWithRetries
         {
             if (($null -eq $RetryableExceptions) -or (Test-RetryableException -Exception $_.Exception -AllowedExceptions $RetryableExceptions))
             {
+                $shouldRetry = $ExceptionRetryEvaluator.Invoke($_.Exception)
```

Comment:
No need to pass the parameter $_.Exception since you're not using it anywhere for any ExceptionRetryEvaluator

Reply:
Using it in next PR.